### PR TITLE
issue-2-view-posts

### PR DESCRIPTION
**Pull request by josephtnguyen**
chess3
chess4
image

### CODE REVIEW

File: client/pages/join-game.jsx [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+    super(props);
+    this.state = {
+      posts: null
+    };
+  }
+
+  componentDidMount() {
+    const req = {
+      method: 'GET'
+    };
```
Comment (thebearingedge) [inline client/pages/join-game.jsx]: You can leave these out, as the default HTTP method for `fetch` is GET.

File: server/public/styles.css [hunk 2]
```diff
@@ -131,6 +131,38 @@ a:hover {
...
+  border-radius: 5px;
+  height: calc(100vh - 166px);
+  overflow-y: scroll;
+}
+
+.post {
+  background-color: #4e545e;
+  color: #eaecee;
+  width: 100%;
+  height: auto;
```
Comment (thebearingedge) [inline server/public/styles.css]: Is `height: auto;` required? Honest question

File: server/public/styles.css [hunk 3]
```diff
@@ -131,6 +131,38 @@ a:hover {
...
+.post:hover {
+  background-color: #1a1c20;
+}
+
+.post-board-container {
+  position: relative;
+  width: 100%;
+  height: 0;
+  padding-top: 100%;
+}
```
Comment (thebearingedge) [inline server/public/styles.css:159]: This feels hacky to me. What's going on here?

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, please see my notes. There's some stuff in here I don't quite understand. Also, let's fix your post form real quick:

- your text area label should be "Message" instead of "message:" (no colon, capitalized)
- Add 0.25rem of padding to the text area, so its input value isn't crammed against its border

File: server/public/styles.css (hunk 3 above)
Comment (josephtnguyen) [reply server/public/styles.css:159]: https://tylerduprey-52451.medium.com/a-perfect-square-with-css-964499440998
This was what I found as a solution to a "responsive square".  There were many similar articles, but I thought this one was the most thorough.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**